docs/quickstart.html

```diff
@@ -25,6 +25,17 @@ <h2>Shell completion</h2>
     <p>The script will complete subcommands and flags but most relevantly can also complete
       build labels by reading BUILD files for you.</p>
 
+    <h2>Buildfile Language Protocol Server(LPS)</h2>
+    <p>Please ships with a language server for build files. LPS follows standard
+        <a href="https://langserver.org/">language server protocol</a>
+    from VS Code.</p>
+
+    <p>To use LPS, simply download the language server protocol plugin for your favorite editor,
+    and run the binary at <code>~/.please/build_langserver</code></p>
```

Comment:
this should probably suggest `plz tool langserver` rather than the direct location?

Reply:
In that case, I will make it `plz tool lps` :)